be/src/exec/parquet/utils.cpp

```diff
@@ -10,8 +10,9 @@ CompressionTypePB convert_compression_codec(tparquet::CompressionCodec::type cod
         return NO_COMPRESSION;
     case tparquet::CompressionCodec::SNAPPY:
         return SNAPPY;
+    // parquet-mr uses hadoop-lz4. more details refers to https://issues.apache.org/jira/browse/PARQUET-1878
     case tparquet::CompressionCodec::LZ4:
-        return LZ4;
+        return LZ4_HADOOP;
```

Comment:
If this parquet file is not generated by Hadoop, can this file be parsed properly after this PR?

Reply:
Yep, I will fall back to lz4. 

https://github.com/StarRocks/starrocks/pull/3222/files#diff-6d4580b48dea040705eab7a993d69c6ebc36ae8caaeeab3a5e9967abc72da1bfR168